data/STRchive-database.json

```diff
@@ -3855,10 +3705,98 @@
         "tr_gnomad":[
             "TR51895"
         ],
-        "disease_description":"Cortical tremor, seizures with generalised motor (tonic-clonic) onset (PMID: 38876750); Adult-onset cortical tremor with epilepsy (PMID: 39349043)",
+        "disease_description":"Cortical tremor, seizures with generalised motor (tonic-clonic) onset [@pmid:38876750]; Adult-onset cortical tremor with epilepsy [@pmid:39349043].",
         "locus_tags":[],
         "disease_tags":[]
     },
+    {
+
+   "chrom":"chr15",
+       "start_hg38":156591766,
+       "stop_hg38":156591783,
+       "start_hg19":156561558,
+       "stop_hg19":156561575,
+       "start_t2t":155728132,
+       "stop_t2t":155728149,
+       "id":"NME_NAXE",
+       "disease_id":"NME",
+       "gene_strand":"+",
+       "reference_motif_reference_orientation":[
+           "GGGCC"
+       ],
+       "pathogenic_motif_reference_orientation":[
+           "GGGCC"
+       ],
+       "pathogenic_motif_gene_orientation":[
+           "GGGCC"
+       ],
+       "benign_motif_reference_orientation":[],
+       "benign_motif_gene_orientation":[],
+       "unknown_motif_reference_orientation":[],
+       "unknown_motif_gene_orientation":[],
+       "disease":"NAXE-related mitochondrial encephalopathy",
+       "gene":"NAXE",
+       "flank_motif":null,
+       "locus_structure":"(GGGCC)*",
+       "inheritance":[
+           "AR"
+       ],
+       "type":"5' UTR",
+       "location_in_gene":null,
+       "benign_min":2.0,
+       "benign_max":7.0,
+       "intermediate_min":null,
+       "intermediate_max":null,
+       "pathogenic_min":200.0,
+       "pathogenic_max":200.0,
+       "ref_copies":null,
+       "motif_len":5,
+       "age_onset":"Predominately infantile onset, extending to 20y or older for general disease; proband onset at one year [@doi:10.1038/s41525-024-00429-5].",
```

Comment:
I'd reorder to emphasis what is known about the TR disease specifically first. E.g. something like: 
Single repeat expansion case had age of onset at 1 year, 1 month. For NAXE-related mitochondrial encephalopathy more generally, predominately infantile onset, extending to 20y or older.